web/pages/image-page/Card.tsx

```diff
@@ -1,10 +1,34 @@
 import { type JSX, useEffect, useState } from 'react'
 import { FluentChevronDown20Regular } from '../../components/icons/fluent-chevron-down-20-regular'
 import { FluentChevronUp20Regular } from '../../components/icons/fluent-chevron-up-20-regular'
+import { FluentOpen16Regular } from '../../components/icons/fluent-open-16-regular'
+
+export type TabActionProps = {
+  action: TabAction
+}
+
+export function TabAction({ action }: TabActionProps): JSX.Element | undefined {
+  switch (action.type) {
+    case 'external-link':
+      return (
+        <a
```

Comment:
Tab index, hover and focus styles are needed to align with usability goals and the look of other buttons and links.